lib/applications.js

```diff
@@ -18,16 +18,24 @@ const errors = require('./errors');
 //                   has the following signature
 //                   (connection, <0 or more event arguments>)
 //   version - application version of 'name' application (optional)
+//   sessionStorageProvider - provider for session storage (optional).
+//                            If provided will be used to store sessions
```

Comment:
I feel like the subject is missing in this sentence. How about "If provided, it will..."?